applications/notifications/server/notifications/services/event_service.py

```diff
@@ -0,0 +1,97 @@
+import json
+import time
+
+from cloudharness import log
+from cloudharness.events.client import EventClient
+from cloudharness.utils.config import CloudharnessConfig as conf
+from kafka.errors import TopicAlreadyExistsError
+from types import SimpleNamespace as Namespace
+
+from notifications.services.notification_service import notify
+
+
+class NotificationHandler:
+    def __init__(self, event_type, app_name, message_type, events):
+        self.message_type = message_type
+        self.events = events
+        self.topic_id = f"{app_name}.{event_type}.{message_type}"
+
+    def handle_event(self, message):
+        """
+        Handle the received event
+
+        Args:
+            message: the message
+        """
+        if self.message_type == message.get("message_type"):
+            operation = message.get("operation")
+            for event in self.events:
+                if event == operation:
+                    meta = message.get("meta", {})
+                    app_name = meta.get("app_name", "")
+                    obj_id = message.get("uid")
```

Comment:
I understand your use case, "uid" is an optional attribute.
For cdc it is "very nice to have" a uid separate from the object. E.g. in case we delete the object, then we don't have the object properties and thus we need a way to identify the object. The "uid" attribute contains the object unique identifier.

Reply:
I understand that the uid is important in CRUD logic. If we want to make this library a special case for CRUD messages, I think that it should be clear from the api that this is the purpose. If it's a generic event handling capability  (which actually the implementation is) then parameters should be generic in the same way.

Reply:
agree in this app we can ignore the uid, removed it